src/main/resources/db/migration/V1__init.sql

```diff
@@ -1 +1,17 @@
+create table variants
+(
+    id    bigserial primary key,
+    chrom varchar not null,
+    pos   bigint  not null,
+    ref   varchar,
+    alt   varchar,
+    unique (chrom, pos, ref, alt)
```

Comment:
please add new unique non-nullable varchar column `variant_code` and create unique index for it.

Reply:
added